openshift-pipelines-client.spec

```diff
@@ -0,0 +1,81 @@
+%global debug_package %{nil}
+%global package_name openshift-pipelines-client
+%global product_name OpenShift Pipelines
+%global golang_version 1.12
+%global tkn_version 1.0.0
+%global tkn_release 1
+%global tkn_cli_version v%{tkn_version}
+%global repo github.com/openshift/tektoncd-cli
+%global source_dir tekton-client-%{tkn_version}-%{tkn_release}
+%global source_tar %{source_dir}.tar.gz
+
+Name:           %{package_name}
+Version:        %{tkn_version}
+Release:        %{tkn_release}
+Summary:        A command line tool to enhance users productivity for working with %{product_name}
+License:        ASL 2.0
+URL:            https://%{repo}/releases/tag/v%{version}
+
+Source0:        %{source_tar}
+BuildRequires:  gcc
+BuildRequires:  golang >= %{golang_version}
```

Comment:
@hrishin : I think we need following here
```
Provides:    %{package_name}
Obsoletes:   %{package_name}
```

Reply:
@navidshaikh :+1: